paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -662,11 +594,17 @@ internal class PaymentSheetActivityTest {
 
     @Test
     fun `google pay flow updates the scroll view before and after`() {
-        val scenario = activityScenario()
+        val viewModel = createViewModel(isGooglePayAvailable = true)
+        val scenario = activityScenario(viewModel)
+
         scenario.launch(intent).onActivity { activity ->
             viewModel.checkoutIdentifier = CheckoutIdentifier.SheetTopGooglePay
+            idleLooper()
```

Comment:
Should these be consistent? `composeTestRule.waitForIdle`? If no, why not?

Reply:
Yes, I guess they should. Updated accordingly.